shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeSyncService.java

```diff
@@ -0,0 +1,51 @@
+package com.redhat.service.smartevents.shard.operator.v2;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.smartevents.infra.v2.api.models.OperationType;
+import com.redhat.service.smartevents.infra.v2.api.models.dto.BridgeDTO;
+import com.redhat.service.smartevents.shard.operator.v2.converters.ManagedBridgeConverter;
+import com.redhat.service.smartevents.shard.operator.v2.providers.NamespaceProvider;
+import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedBridge;
+
+@ApplicationScoped
+public class ManagedBridgeSyncService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ManagedBridgeSyncService.class);
+
+    @Inject
+    ManagerClient managerClient;
+
+    @Inject
+    ManagedBridgeService managedBridgeService;
+
+    @Inject
+    NamespaceProvider namespaceProvider;
+
+    public void syncManagedBridgeWithManager() {
+        managerClient.fetchBridgesToDeployOrDelete()
+                .onItem()
+                .invoke(this::processDelta)
+                .subscribe().with(
+                        success -> LOGGER.debug("Successfully processed ManagedBridge deltas"),
+                        failed -> LOGGER.debug("Failed to process ManagedBridge deltas"));
```

Comment:
Please use `error` for this instead of `debug`

Reply:
done